client/my-sites/checkout/checkout-thank-you/jetpack-activation-instructions.tsx

```diff
@@ -0,0 +1,61 @@
+import { Button } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import ExternalLink from 'calypso/components/external-link';
+
+const JetpackActivationInstructions: React.FC = () => {
+	const translate = useTranslate();
+
+	return (
+		<>
+			<p>{ translate( "If you don't have Jetpack installed, follow these instructions:" ) }</p>
+
+			<ul className="licensing-thank-you-manual-activation-instructions__list">
```

Comment:
Since this is an ordered list, this could be an `ol` tag.

Reply:
Nice catch @monsieur-z , completely missed this one. thanks 👍